test/data/delta/delta_1.1.0_delta_with_param.py

```diff
@@ -7,13 +7,10 @@
 class CreateViews(DeltaPy):
 
     def run(self):
-        my_field_length = self.variable('my_field_length')
-
         conn = psycopg2.connect("service={0}".format(self.pg_service))
         cursor = conn.cursor()
-        sql = "ALTER TABLE northwind.customers ALTER COLUMN country TYPE varchar({l});".format(l=my_field_length)
+        sql = ("ALTER TABLE northwind.customers "
+               "ALTER COLUMN country TYPE varchar(%(my_field_length)s)")
```

Comment:
I get your point here, but the demonstration here is more how to access variables and use them on Python side. The test might not be the best ever as it's not the recommended way to pass variables to query, I would still prefer to keep that on the Python side.

Reply:
I don't like the idea of having code that shows bad practices. And relying on psycopg2 for passing parameters is consistent with the way it works with SQL deltas.

Reply:
Actually I am not sure the way I've done is correct either, as `my_field_length` is to be used as an SQL identifier rather than an SQL value.

I'll remove that commit from that PR, and I will possibly open a separate PR for this particular issue.

Reply:
Done.